apps/dashboard/config/initializers/rclone.rb

```diff
@@ -3,10 +3,21 @@
 Rails.application.config.after_initialize do
   next unless Configuration.remote_files_enabled?
 
-  remotes = RcloneUtil.list_remotes.map { |r| FavoritePath.new('', title: r, filesystem: r) }
+  remotes = RcloneUtil.list_remotes
+
+  if Configuration.remote_validation?
```

Comment:
Is there a reason why you'd want invalid paths?

Reply:
Not really, I mostly added that incase someone is concerned about slow logins to OOD. For us, removing the configuration option would be fine.